<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendRecvDialog</class>
 <widget class="QDialog" name="SendRecvDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>1123</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Record Signal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle:horizontal {
margin: 4px 0px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
stop:0 rgba(255, 255, 255, 0), 
stop:0.5 rgba(100, 100, 100, 100), 
stop:1 rgba(255, 255, 255, 0));
image: url(:/icons/icons/splitter_handle_vertical.svg);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>658</width>
         <height>1111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
 border: none;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="19" column="0">
            <widget class="QProgressBar" name="progressBarMessage">
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v/%m</string>
             </property>
            </widget>
           </item>
           <item row="18" column="0">
            <widget class="QLabel" name="labelCurrentMessage">
             <property name="text">
              <string>Current message:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="lReceiveBufferFullText">
             <property name="text">
              <string>Receive buffer full:</string>
             </property>
            </widget>
           </item>
           <item row="21" column="0">
            <widget class="QProgressBar" name="progressBarSample">
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v/%m</string>
             </property>
            </widget>
           </item>
           <item row="20" column="0">
            <widget class="QLabel" name="lSamplesSentText">
             <property name="text">
              <string>Current sample:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="lTimeText">
             <property name="text">
              <string>Time (in seconds):</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lSamplesCapturedText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Samples captured:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="lSignalSizeText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Signal size (in MiB):</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="lSamplesCaptured">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QLabel" name="lTime">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QLabel" name="lSignalSize">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QLabel" name="lblRepeatText">
             <property name="text">
              <string>Current iteration:</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="QLabel" name="lblCurrentRepeatValue">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="labelReceiveBufferFull">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnStart">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Record signal</string>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
               <property name="icon">
                <iconset theme="media-record">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnStop">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Stop recording</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
               <property name="icon">
                <iconset theme="media-playback-stop">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnSave">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string>Save...</string>
               </property>
               <property name="icon">
                <iconset theme="document-save">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnClear">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Clear</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset theme="view-refresh">
                 <normaloff>.</normaloff>.</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtEditErrors">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="page_receive">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="LiveGraphicView" name="graphicsViewReceive">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_send">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="EditableGraphicView" name="graphicsViewSend">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="renderHints">
              <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
             </property>
             <property name="transformationAnchor">
              <enum>QGraphicsView::NoAnchor</enum>
             </property>
             <property name="resizeAnchor">
              <enum>QGraphicsView::NoAnchor</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Hint: You can edit the raw signal before sending.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_continuous_send">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="LiveGraphicView" name="graphicsViewContinuousSend">
             <property name="renderHints">
              <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_spectrum">
          <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1">
           <item>
            <widget class="LiveGraphicView" name="graphicsViewFFT"/>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicsViewSpectrogram">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="renderHints">
              <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
             </property>
             <property name="viewportUpdateMode">
              <enum>QGraphicsView::MinimalViewportUpdate</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_sniff">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="LiveGraphicView" name="graphicsView_sniff_Preview"/>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="txtEd_sniff_Preview">
             <property name="lineWrapMode">
              <enum>QPlainTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="maximumBlockCount">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAccept">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accept the sniffed data and load it into &lt;span style=&quot; font-weight:600;&quot;&gt;Analysis&lt;/span&gt; tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Accept data (Open in Analysis)</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_y_scale">
           <property name="text">
            <string>Y-Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderYscale">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LiveGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>urh.ui.views.LiveGraphicView.h</header>
  </customwidget>
  <customwidget>
   <class>EditableGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>urh.ui.views.EditableGraphicView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnStart</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>txtEd_sniff_Preview</tabstop>
  <tabstop>sliderYscale</tabstop>
  <tabstop>txtEditErrors</tabstop>
  <tabstop>graphicsViewSend</tabstop>
  <tabstop>graphicsViewReceive</tabstop>
  <tabstop>btnAccept</tabstop>
 </tabstops>
 <resources>
  <include location="urh.qrc"/>
 </resources>
 <connections/>
</ui>
